app/src/main/res/xml/pref_migrating.xml

```diff
@@ -0,0 +1,21 @@
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <com.kabouzeid.appthemehelper.common.prefs.supportv7.ATEPreferenceCategory android:title="@string/pref_header_migrating">
+
+        <com.poupa.vinylmusicplayer.preferences.ExportSettingsPreference
+            app:iconSpaceReserved="false"
+            android:key="export_settings"
+            android:defaultValue="1m"
+            android:title="@string/pref_title_export" />
+
+        <com.kabouzeid.appthemehelper.common.prefs.supportv7.ATEPreference
+            app:iconSpaceReserved="false"
+            android:key="import_settings"
+            android:defaultValue="1m"
+            android:title="@string/pref_title_import" />
```

Comment:
I'd add some description text here as well, in addition to the title.
For instance: "Import the settings from previously exported file"